sunpy/visualization/visualization.py

```diff
@@ -45,15 +48,51 @@ def axis_labels_from_ctype(ctype, unit):
     labels = {'HGLN': 'Heliographic Longitude',
               'CRLN': 'Carrington Longitude',
               'HPLN': 'Helioprojective Longitude (Solar-X)',
+              'HRLN': 'Helioprojective Position Angle',
               'SOLX': 'Heliocentric X',
 
               'HGLT': 'Latitude',
               'CRLT': 'Latitude',
               'HPLT': 'Helioprojective Latitude (Solar-Y)',
+              'HRLT': 'Helioprojective Declination',
               'SOLY': 'Heliocentric Y'}
 
     label = labels.get(ctype_short, f"{ctype}")
     if unit is not None:
         label += f' [{unit}]'
 
     return label
+
+
+def show_hpr_impact_angle(declination_axis):
```

Comment:
I have got to have a think about how to patch astropy so we don't have to use the private API here.

Reply:
That's why I was interested in passing a matplotlib formatter into `set_major_formatter()`, but that turned into a whole mess because of how WCSAxes does things.  Even if some change is made to astropy, in the near term we still ought to provide users with an approach that works on older versions of astropy.

Reply:
Yup, definitely